recipes/megaman/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# On OSX, we need to ensure we're using conda's gcc/g++
+# This is because flann (one of the build dependencies) uses OpenMP
+if [[ `uname` == Darwin ]]; then
```

Comment:
Ah! Sorry. Missed the note.

Yes, we must use gcc. There has been some brief discussion about taking a newer `clang` system compiler for OS X (that has OpenMP support) and compiling it on an old OS X (Apple provides the source code) so that we can remain compatible on old systems, but supply the latest features. A similar strategy is already being pursued on Linux. If we do this, we may not require `gcc` for these situations in the future. Actually, the only required thing from the GCC toolkit will be `gfortran` if we have Fortran code.